src/storage/hummock_sdk/src/lib.rs

```diff
@@ -266,3 +267,55 @@ pub fn version_checkpoint_path(root_dir: &str) -> String {
 pub fn version_checkpoint_dir(checkpoint_path: &str) -> String {
     checkpoint_path.trim_end_matches(|c| c != '/').to_string()
 }
+
+#[derive(Clone, Copy, PartialEq, Eq, Hash, Default, Debug, PartialOrd, Ord)]
+pub struct EpochWithGap(u64);
+
+impl EpochWithGap {
+    pub fn new(epoch: u64, spill_offset: u64) -> Self {
+        #[cfg(not(feature = "enable_test_epoch"))]
```

Comment:
Can we have the following code to avoid having the `enable_test_epoch` enabled in release mode?
```
#[cfg(all(feature = "enable_test_epoch", not(debug_assertions)))]
{
    compile_error!("enable_test_epoch only allowed in debug mode")
}
```

Reply:
Yes, next pr for sure.

Reply:
It's just 3 lines of change. It's fine to be included in this PR.